src/backend/src/services/design-reviews.services.ts

```diff
@@ -0,0 +1,97 @@
+import { Design_Review, Design_Review_Status, User } from '@prisma/client';
+import { isNotLeadership } from 'shared';
+import prisma from '../prisma/prisma';
+import { NotFoundException, AccessDeniedMemberException, DeletedException, HttpException } from '../utils/errors.utils';
+import { getUsers, getUserPrismaIds } from '../utils/users.utils';
+import { validateMeetingTimes } from '../utils/design-reviews.utils';
+export default class DesignReviewsService {
+  /**
+   * Edits a Design_Review in the database
+   * @param user the user editing the design review (must be leadership)
+   * @param designReviewId the id of the design review to edit
+   * @param dateScheduled the date of the design review
+   * @param teamTypeId the team that the design_review is for (software, electrical, etc.)
+   * @param requiredMembers required members for the design review
+   * @param optionalMembers optional members for the design review
+   * @param isOnline is the design review online (IF TRUE: zoom link should be requried))
+   * @param isInPerson is the design review in person (IF TRUE: location should be required)
+   * @param zoomLink the zoom link for the design review meeting
+   * @param location the location for the design review meeting
+   * @param docTemplateLink the document template link for the design review
+   * @param status see Design_Review_Status enum
+   * @param attendees the attendees for the design review (should they have any relation to the other shit / can't edit this after STATUS: DONE)
+   * @param meetingTimes meeting time must be between 0-48 (9-9, 15 minute increments)
+   */
+
+  static async editDesignReviews(
+    user: User,
+    designReviewId: string,
+    dateScheduled: Date,
+    teamTypeId: string,
```

Comment:
you should ensure that the teamTypeId corresponds to an existing teamType in the database